core/vm/instructions.go

```diff
@@ -829,6 +829,22 @@ func opSelfdestruct(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext
 	return nil, errStopToken
 }
 
+func opSelfdestruct6780(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
+	if interpreter.readOnly {
+		return nil, ErrWriteProtection
+	}
+	beneficiary := scope.Stack.pop()
+	balance := interpreter.evm.StateDB.GetBalance(scope.Contract.Address())
+	interpreter.evm.StateDB.SubBalance(scope.Contract.Address(), balance)
+	interpreter.evm.StateDB.AddBalance(beneficiary.Bytes20(), balance)
+	interpreter.evm.StateDB.SendAll(scope.Contract.Address())
```

Comment:
The way this is implemented, if `sendall` is executed many times, it will re-sendall. Just noting it, not sure if it's explicitly stated in the spec. 

* A sends 100 wei to b
* A -> b.Sendall( c) // sends 100 to c
* A sends 100 wei to b
* A -> b.Sendall(c) // sends 100 to c again

Reply:
Good observation.  This behavior seems in-line with the spec IMO.